pkgs/top-level/all-packages.nix

```diff
@@ -2309,6 +2309,8 @@ in
     buildGoModule = buildGo112Module;
   };
 
+  circus = python3Packages.callPackage ../tools/networking/circus { };
```

Comment:
drop the `python3Packages.`

Reply:
without it `called without required argument 'buildPythonApplication'` appears